README.md

```diff
@@ -74,8 +80,14 @@ this method should set the pizza's temperature to be 300. Note: you may not use
 the variable pizza inside your method.
 
 Your Answer:
-```js
-// write code here
+```
+var pizza = {
+  temp: "70",
+  bake: function(newTemp){
+    console.log("temp has been set to 300");
+    pizza.temp = "300";
```

Comment:
without using the `pizza` variable inside the object, how could we access the object's `temp` property? hint: using what we learned about context